homeassistant/components/venstar/__init__.py

```diff
@@ -111,22 +112,11 @@ def __init__(
         """Initialize the data object."""
         super().__init__(venstar_data_coordinator)
         self._config = config
-        self._update_attr()
-        self.coordinator = venstar_data_coordinator
-
-    @property
-    def _client(self):
-        """Return the venstar client."""
-        return self.coordinator.client
-
-    @callback
-    def _update_attr(self) -> None:
-        """Update the state and attributes."""
+        self._client = self.coordinator.client
```

Comment:
We'll save an attribute access.
```suggestion
        self._client = venstar_data_coordinator.client
```